eng/scripts/verify-npm-tags.ps1

```diff
@@ -0,0 +1,59 @@
+param (
+  [Parameter(mandatory = $true)]
+  $artifactName,
+  [Parameter(mandatory = $true)]
+  $workingDirectory
+)
+
+# TODO: delete testing comments below
+#'@azure/eventhubs-checkpointstore-table' alpha, no beta, no GA
+#'@azure/openai' alpha, beta, no GA
+#'@azure/video-indexer-widgets' no alpha, no beta, GA
+#'@azure/template' alpha, beta, GA
+#$pkgProps.PackageId
+
+. (Join-Path $PSScriptRoot "../common/scripts/common.ps1")
+
+$packageProperties = Get-PkgProperties -PackageName $artifactName
```

Comment:
This function will not be available when we move to tools eng/common

Reply:
this function is under eng/common/scripts/Package-Properties.ps1 though
https://github.com/Azure/azure-sdk-for-js/blob/main/eng/common/scripts/Package-Properties.ps1#L147

Reply:
follow up comments below